tools/c7n_org/README.md

```diff
@@ -92,10 +117,33 @@ groups of policies via their tags with `-l`.
 
 See `c7n-org run --help` for more information.
 
-# Other commands
+## Other commands
 
-c7n-org also supports running arbitrary scripts against accounts via the run command, which
+c7n-org also supports running arbitrary scripts on AWS against accounts via the run-script command, which
 exports standard AWS SDK credential information into the process environment before executing.
 
 c7n-org also supports generating reports for a given policy execution across accounts via
 the `c7n-org report` subcommand.
+
+## Additional Azure Instructions
+
+In order for Cloud Custodian to have access to your subscription, permission must be 
```

Comment:
This needs clarification in that it only applies if your using a service account. If your using user credentials, that its not per se accurate. ie. the implication is that custodian only runs with a service account, when thats one mode of authentication. Ie better phrasing `If your using custodian with service account credentials,`

Reply:
i'll address this in a followup i'm doing for gcp support

Reply:
hmmm.. actually this content is better placed in the azure docs section thats linked to from here.